### ISSUE TITLE
Calling an <svg> in the header violates 508c

### ISSUE THREAD

**Issue by jazzsequence** [WebDevStudios/wd_s#108]
https://github.com/WebDevStudios/wd_s/blob/master/inc/scripts.php#L110

Also the `display: none;` throws a warning.

![total_validator_results](https://cloud.githubusercontent.com/assets/991511/11134011/e36d723e-8956-11e5-89bf-344712f9972e.png)

Should we put the `<svg>` call in the footer? After wp_head?

**Comment by gregrickaby** [comment]
I believe this has to go in the header in order for us to use `<svg>` like sprites. Will need to research this.

**Comment by gregrickaby** [comment]
On my local, I moved the SVG definitions to the footer and ran a [CBT test](http://app.crossbrowsertesting.com/public/i94e9360db902757/screenshots/z5ff4e4e34921d4b3be7). Unfortunately, moving the SVG definitions to the footer will break SVGs in Safari 8 and Safari for iOS 8. 

Then I moved the SVG definitions immediately after `<body>` and re-ran a [CBT test](http://app.crossbrowsertesting.com/public/i94e9360db902757/screenshots/z3d3c63c7fc8ace5f2e4) and everything looks OK (except Safari 7). It's clear Safari needs the definitions to be defined before it can render the SVG. Ran it through TotalValidator (US Section 508) and this was error free. It does however, still [throw a warning](https://dl.dropbox.com/s/2q5wcmnshybv5ct/Screenshot%202015-11-16%2009.08.36.png?dl=0) (which is fine).

For comparison, here is a [CBT test](http://app.crossbrowsertesting.com/public/i94e9360db902757/screenshots/z7082f089d78d989a2a1) with SVG definitions in `<head>`. (Safari 7 is broken as well).

For reference: "Use SVG as sprites" idea came from: https://css-tricks.com/svg-sprites-use-better-icon-fonts/

When Safari 10 is released, hopefully, we can move these to the footer. (WDS supports the last two browsers)

**tl;dr:** _Moving SVG defs immediately after `<body>` allows us to pass 508c compliance while still retaining our normal SVG workflow._

**Comment by jazzsequence** [comment]
That was going to be my next suggestion. :+1:

**Comment by jazzsequence** [comment]
> It does however, still throw a warning

I'm planning to submit a PR this week to add a class so those style warnings can be taken care of with the theme stylesheet. There's no reason to use inline CSS if we don't need it.
